eventmesh-runtime/src/main/java/org/apache/eventmesh/runtime/core/protocol/http/consumer/ConsumerManager.java

```diff
@@ -73,100 +73,97 @@ public void init() throws Exception {
-                                            .put(consumerGroup, consumerGroupConf);
-                                        logger.info(
-                                            "consumerGroup {} client info changed, "
-                                                + "consumerGroupConf {}", consumerGroup,
-                                            JsonUtils.serialize(consumerGroupConf));
-                                        try {
-                                            notifyConsumerManager(consumerGroup, consumerGroupConf);
-                                        } catch (Exception e) {
-                                            e.printStackTrace();
+                                            .get(consumerGroup);
+                                    Map<String, ConsumerGroupTopicConf> map =
+                                        consumerGroupConf.getConsumerGroupTopicConf();
+                                    for (String topicKey : map.keySet()) {
+                                        if (StringUtils.equals(topic, topicKey)) {
+                                            ConsumerGroupTopicConf latestTopicConf =
+                                                new ConsumerGroupTopicConf();
+                                            latestTopicConf.setConsumerGroup(consumerGroup);
+                                            latestTopicConf.setTopic(topic);
+                                            latestTopicConf.setSubscriptionItem(
+                                                map.get(topicKey).getSubscriptionItem());
+                                            latestTopicConf.setUrls(clientUrls);
+
+                                            latestTopicConf.setIdcUrls(idcUrls);
+
+                                            map.put(topic, latestTopicConf);
                                         }
                                     }
-
-                                } else {
-                                    logger.info("consumerGroup {} client info removed",
-                                        consumerGroup);
-                                    //remove
+                                    eventMeshHTTPServer.localConsumerGroupMapping
+                                        .put(consumerGroup, consumerGroupConf);
+                                    logger.info(
+                                        "consumerGroup {} client info changed, "
+                                            + "consumerGroupConf {}", consumerGroup,
+                                        JsonUtils.serialize(consumerGroupConf));
                                     try {
-                                        notifyConsumerManager(consumerGroup, null);
+                                        notifyConsumerManager(consumerGroup, consumerGroupConf);
                                     } catch (Exception e) {
                                         e.printStackTrace();
```

Comment:
Please use logger to optimize this print, this can avoid the .out file expansion.